xdocs/usermanual/properties_reference.xml

```diff
@@ -602,6 +602,23 @@ JMETER-SERVER</source>
 </property>
 </properties>
 </section>
+
+  <section name="&sect-num;.14 HTTP Header Manager configuration" anchor="http_header_manager">
```

Comment:
Note that `&sect-num;.14` duplicates the number of the previous section, however, I believe we'll remove section numbers in https://github.com/apache/jmeter/issues/5915